src/transformers/models/auto/tokenization_auto.py

```diff
@@ -273,6 +273,7 @@
         ("ovis2", "Qwen2TokenizerFast" if is_tokenizers_available() else None),
         ("owlv2", "CLIPTokenizerFast" if is_tokenizers_available() else None),
         ("owlvit", "CLIPTokenizerFast" if is_tokenizers_available() else None),
+        ("paddleocr_vl", "LlamaTokenizer" if is_tokenizers_available() else None),
```

Comment:
```suggestion
        ("paddleocr_vl", "TokenizersBackend" if is_tokenizers_available() else None),
```
from looking at the `tokenizer.json` on the hub, its not a llama!
```python
"normalizer": {
    "type": "Sequence",
    "normalizers": [
      {
        "type": "Replace",
        "pattern": {
          "String": " "
        },
        "content": "▁"
      }
    ]
  },
  "pre_tokenizer": null,
```

while llama would initialize: 
```python 
        self._tokenizer.normalizer = None
        self._tokenizer.pre_tokenizer = pre_tokenizers.Metaspace(
            replacement="▁", prepend_scheme=_get_prepend_scheme(self.add_prefix_space, self), split=False
        )
```